tests/apps/core/test_authentication_delegation.py

```diff
@@ -41,19 +38,14 @@ def test_user_menu_with_profile_urls(self):
         response = self.client.get(page.get_public_url())
         self.assertContains(response, "richie-react richie-react--user-login")
 
-        pattern = r".*data-props=.*{}.*{}.*".format(
-            "Profile", "https://richie.education:9999/profile"
-        )
-
+        pattern = r".*data-props=.*Profile.*https://richie\.education:9999/profile.*"
         self.assertIsNotNone(re.search(pattern, str(response.content)))
         self.assertContains(
             response,
             (
                 r"\u0022authentication\u0022: "
-                r"{{\u0022endpoint\u0022: \u0022{}\u0022, \u0022backend\u0022: \u0022{}\u0022}}"
-            ).format(
-                "https://richie.education:9999",
-                "richie.apps.courses.lms.base.BaseLMSBackend",
+                r"{{\u0022endpoint\u0022: \u0022https://richie.education:9999\u0022, "
+                r"\u0022backend\u0022: \u0022richie.apps.courses.lms.base.BaseLMSBackend\u0022}}"
```

Comment:
This test currently fails.
```suggestion
                r"{\u0022endpoint\u0022: \u0022https://richie.education:9999\u0022, "
                r"\u0022backend\u0022: \u0022richie.apps.courses.lms.base.BaseLMSBackend\u0022}"
```

Format silently removed the double `{` & `}`...

Reply:
Yep. I reverted it...